### ISSUE TITLE
Translated time unit causes a fatal error

### ISSUE THREAD

**Issue by mintplugins** [RebelCode/edd-bookings#26]
Hey Bookings team, 

A customer has opened a Ticket with bookings reporting this:
```
After I have translated the special word "time" into the PO file for EDD booking, I get this error message next time I edit and save a download.

Fatal error: Call to undefined method Aventura\Diary\DateTime\Duration::timmar() in /storage/content/24/201524/bokningsnavet.se/public_html/wp-content/plugins/edd-bookings/includes/CustomPostType/ServicePostType.php on line 246.
```

Do you know why that might be happening for them?

**Comment by mecha** [comment]
Yeah. It looks like the plugin is incorrectly using the translated time unit string, rather than the constant value for calculated duration. Whoops!

We'll get this fixed in the next maintenance release. Thanks for the report.

**Comment by mintplugins** [comment]
Thanks

**Comment by mintplugins** [comment]
@mecha Hey there. Just checking in on this. We have a customer who is growing very impatient on this. Do you know if any progress has been made on this or if I can give them any type of a timeframe to expect?

**Comment by mecha** [comment]
@mintplugins You can expect a release some time next week.

**Comment by mecha** [comment]
Fixed in [2.2.4](https://github.com/RebelCode/edd-bookings/releases/tag/v2.2.4),
